components/components_CSS/Badge/badge.css

```diff
@@ -0,0 +1,95 @@
+
+@import url("/root.css");
+
+img{
+    max-width: 100%;
+    max-height: 100%;
+    
+ }
+
+ /* -------------------------------------avatar styling */
+.avatar{
+    position: relative;
+
+}
+
+.avater-extra-large{
+   max-height: 15rem;
+    max-width: 15rem;
+
+}
+
+.avatar-large{
+    max-height: 12rem;
+    max-width: 12rem;
+}
+
+.avatar-medium{
+    max-height: 8rem;
+    max-width: 8rem;
+  
+}
+
+.avatar-small{
+    max-height: 5rem;
+    max-width: 5rem;
+}
+
+
+.avatar-extra-small{
+    max-height: 3rem;
+    max-width: 3rem;
+
+}
+
```

Comment:
Why have you put avatar css in badge css??

Reply:
Actually I am adding badges to different avatar, so used avatar css